cmd/notary/tuf.go

```diff
@@ -348,7 +358,50 @@ func tufRemove(cmd *cobra.Command, args []string) {
 	saveRepo(repo, filestore)
 }
 
-func verify(cmd *cobra.Command, args []string) {
+func execCmd(cmd *cobra.Command, args []string) {
+	sh := os.Getenv("SHELL")
+	if sh == "" {
+		sh = "/bin/sh"
+	}
+	if _, err := os.Stat(sh); os.IsNotExist(err) {
+		fatalf("unable to determine shell, specify SHELL in environment")
+	}
+	shCmd := exec.Command(sh)
+	shCmd.Stdout = os.Stdout
+	shCmd.Stderr = os.Stderr
+	shStdin, err := shCmd.StdinPipe()
+	if err != nil {
+		logrus.Error("unable to create stdin pipe:", err)
+		return
+	}
+	go verify(cmd, args, shStdin)
```

Comment:
Doesn't this cause a race condition where a command could run before it is verified?